test/integration/generated/head/Source/AutoRestHeadTestClient.swift

```diff
@@ -254,8 +254,8 @@ public final class AutoRestHeadTestClient: PipelineClient {
                     return
                 }
                 if [
-                    204,
-                    404
+                    404,
+                    204
```

Comment:
Perhaps we should sort the status code collection so the generated code is deterministic.